Frameworks/CoreFoundation/URL.subproj/CFURL.c

```diff
@@ -1323,8 +1323,17 @@ static Boolean _stringContainsCharacter(CFStringRef string, UniChar ch) {
     return false;
 }
 
+// WINOBJC : A function for creating percent escapes based on character sets instead of strings has been added. This is used for stringByAddingPercentEscapesUsingEncoding which operates on a valid set of 
```

Comment:
Oh woops, I have a bad habit of letting the formatter handle my comments.

---
In reply to: [77091827](https://github.com/Microsoft/WinObjC/pull/946#discussion_r77091827) [](ancestors = 77091827)